extensions/wikia/AdEngine/js/AdEngine2.js

```diff
@@ -133,6 +133,10 @@ define('ext.wikia.adEngine.adEngine', [
 						eventDispatcher.dispatch('adengine.slot.status', {slotName: slotName, status: 'hop'});
 						tracker.track('hop', adInfo);
 						nextProvider();
+					},
+					renderEnded: function(googleSlot) {
+						log(['renderEnded', provider.name, slotName, googleSlot], 'debug', logGroup);
+						tracker.track('renderEnded', googleSlot);
```

Comment:
In the different hooks we send `adInfo` (something meaningful) to GA, what do we send here (do we want do send it)?